packages/rest/test/unit/rest.component.unit.ts

```diff
@@ -24,22 +24,30 @@ import {
 const SequenceActions = RestBindings.SequenceActions;
 describe('RestComponent', () => {
   describe('Providers', () => {
-    describe('Default implementations are bound', async () => {
-      const app = new Application();
-      app.component(RestComponent);
+    describe('Default implementations are bound', () => {
+      let app: Application;
+      let comp: Component;
 
-      // Stub constructor requirements for some providers.
-      app.bind(RestBindings.Http.CONTEXT).to(new Context());
-      app.bind(RestBindings.HANDLER).to(new HttpHandler(app));
+      before(async () => {
+        app = new Application();
+        app.component(RestComponent);
+
+        // Stub constructor requirements for some providers.
+        app.bind(RestBindings.Http.CONTEXT).to(new Context());
+        app.bind(RestBindings.HANDLER).to(new HttpHandler(app));
+
+        comp = await app.get<Component>('components.RestComponent');
+      });
 
-      const comp = await app.get<Component>('components.RestComponent');
-      for (const key in comp.providers || {}) {
-        it(key, async () => {
-          const result = await app.get(key);
-          const expected: Provider<BoundValue> = new comp.providers![key]();
-          expect(result).to.deepEqual(expected.value());
-        });
-      }
+      it('', async () => {
```

Comment:
Do we get a deprecation warning if we don't wrap this block of code with the empty `it` block?

Reply:
Yes. It's also required to trigger `before` and run multiple tests in a loop.